src/resolvers/rootResolver.test.ts

```diff
@@ -0,0 +1,67 @@
+import { getMockApolloServer } from "../__mocks__/mockApolloServer";
+import { createTestClient, ApolloServerTestClient } from "apollo-server-testing";
+import {gql} from "apollo-server-express";
+import logger from "../utils/logger";
+
+const createRootMutation = gql`
+    mutation {
```

Comment:
Make query and mutation uniform, either make mutation like this
```
mutation Root {
    root
}
```

or, make query like this
```
query {
    root
}
```